tests/test_mcscripter.py

```diff
@@ -0,0 +1,40 @@
+"""
+Tests for converters (so far only pld2sobp.py)
+"""
+import unittest
+import logging
+import pymchelper.utils.mcscripter
+import pymchelper.run
+
+logger = logging.getLogger(__name__)
+
+
+class TestMcScripter(unittest.TestCase):
+    def test_help(self):
+        """ Print usage and exit normally.
+        """
+        try:
+            pymchelper.utils.mcscripter.main([])
```

Comment:
Why do you expect exit code 0 if no arguments to `mcscripter` are provided ?
With argparse tests will fail. Better to check for exit code 2, or check `--version` option